src/transformers/models/timesformer/configuration_timesformer.py

```diff
@@ -0,0 +1,128 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" TimeSformer model configuration"""
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+TIMESFORMER_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "facebook/timesformer": "https://huggingface.co/facebook/timesformer/resolve/main/config.json",
+}
+
+
+class TimesformerConfig(PretrainedConfig):
+    r"""
+    This is the configuration class to store the configuration of a [`TimesformerModel`]. It is used to instantiate a
+    TimeSformer model according to the specified arguments, defining the model architecture. Instantiating a
+    configuration with the defaults will yield a similar configuration to that of the TimeSformer
+    [facebook/timesformer](https://huggingface.co/facebook/timesformer) architecture.
```

Comment:
Does not exist yet.

Reply:
Is there anything I can do about this?

Reply:
I've transferred the checkpoints

Reply:
Thank you for that! However this page still returns 404: https://huggingface.co/facebook/timesformer

Reply:
You should select the URL of the default checkpoint, like https://huggingface.co/facebook/timesformer-base-finetuned-k600

Reply:
@NielsRogge I have updated the related url 👍  Also opened a PR adding related model card: https://huggingface.co/facebook/timesformer-base-finetuned-k600/discussions/1